GoogleUtilities/AppDelegateSwizzler/README.md

```diff
@@ -0,0 +1,151 @@
+# App Delegate Swizzler
+
+## Overview
+
+The App Delegate Swizzler swizzles certain methods on the AppDelegate and allows interested parties (for eg. other SDKs like Firebase Analytics) to register listeners when certain App Delegate methods are called. 
```

Comment:
Keep the line lengths less than 100 so it is readable from and editor as well as markdown.